skfda/exploratory/depth/multivariate.py

```diff
@@ -0,0 +1,33 @@
+import scipy.stats
+
+import numpy as np
+
+from . import outlyingness_to_depth
+
+
+def _stagel_donoho_outlyingness(X, *, pointwise=False):
+
+    if pointwise is False:
+        raise NotImplementedError("Only implemented pointwise")
+
+    if X.ndim_codomain == 1:
+        # Special case, can be computed exactly
+        m = X.data_matrix[..., 0]
+
+        return (np.abs(m - np.median(m, axis=0)) /
+                scipy.stats.median_absolute_deviation(m, axis=0))
```

Comment:
I have the following error while running the example ``examples/plot_magnitude_shape_synthetic.py``:

> AttributeError: module 'scipy.stats' has no attribute 'median_absolute_deviation'

I have the scipy version ``1.1.0`` and it was added in the version ``1.3.0``, we should add to the requirements and maybe increase the coverage.

After update scipy all runs ok.

Reply:
Fixed in 4c53ffa658485efce3bb121c6df1fbb9640bc46b.